### ISSUE TITLE
Get a list of differences

### ISSUE THREAD

**Issue by jycouet** [Microsoft/vsts-node-api#129]
Hello there, 
I found on the API doc a good way to [Get a list of differences](https://www.visualstudio.com/en-us/docs/integrate/api/git/diffs)

Is there something to achive this?

---
This repo is still active?
Thank you for the info 👍

**Comment by bryanmacfarlane** [comment]
Yes, this is still maintained.  It's generated off the product APIs so if it exists in the REST API and C# clients, then it should exist here.

Marking as a question.

**Comment by jycouet** [comment]
Thank you for your answer @bryanmacfarlane 
I just found that interface: `GitCommitDiffs` that looks like what I want, but it's not used in your code.

So yes, I still have the question :)
Thanks

**Comment by bryanmacfarlane** [comment]
Can you log the same issue here?  https://github.com/Microsoft/vsts-dotnet-samples/issues 

That's the C# samples and since this client is generated, it will have the exact same method patterns.

I would log the issue but probably better coming from you.

**Comment by damccorm** [comment]
I've created a PR in the generation code that should add this to the API. Once that's approved, we can regenerate and push here.
